fineract-provider/build.gradle

```diff
@@ -29,6 +29,8 @@ repositories {
 }
 
 project.ext.jerseyVersion = '1.19.4'
+project.ext.mysqlConnector = 'org.mariadb.jdbc:mariadb-java-client'
```

Comment:
how about ditching the _mysql_ prefix from the name here, and just calling this (and the following)... maybe `jdbcArtifact` and `jdbcVersion`?